core/common/buffer.hpp

```diff
@@ -160,4 +173,74 @@ namespace kagome::common {
 
 }  // namespace kagome::common
 
+namespace std {
+
+ /*
+  * std::back_insert_iterator is an output iterator
+  * that appends to a container for which it was constructed.
+  */
+  template <>
+  class back_insert_iterator<kagome::common::Buffer> {
```

Comment:
I know that I told you to put it in header, but after I looked closely, I think this should be in a **cpp** file, because explicit template instantiation is a **definition**, not **declaration**

so, put this in a cpp file and add this cpp file to buffer target